api/lib/application/sco-organization-learners/index.js

```diff
@@ -253,6 +253,67 @@ exports.register = async function (server) {
         tags: ['api', 'sco-organization-learners'],
       },
     },
+    {
+      method: 'POST',
+      path: '/api/schooling-registration-dependent-users/external-user-token',
+      config: {
+        auth: false,
+        handler: scoOrganizationLearnerController.createUserAndReconcileToOrganizationLearnerFromExternalUser,
+        validate: {
+          options: {
+            allowUnknown: false,
+          },
+          payload: Joi.object({
+            data: {
+              attributes: {
+                'campaign-code': Joi.string().empty(Joi.string().regex(/^\s*$/)).required(),
+                'external-user-token': Joi.string().empty(Joi.string().regex(/^\s*$/)).required(),
+                birthdate: Joi.date().format('YYYY-MM-DD').raw().required(),
+                'access-token': Joi.string().allow(null).optional(),
+              },
+              type: 'external-users',
+            },
+          }),
+        },
+        notes: [
+          "Cette route crée un compte utilisateur suite à une connexion provenant d'un IDP externe (GAR). " +
+            "Les informations sont fournies dans un token. Elle réconcilie également cet utilisateur avec l'inscription " +
+            "de l'élève au sein de l'organisation qui a créé la campagne.",
+          "- L'usage de cette route est **dépréciée** en faveur de /api/sco-organization-learners/external",
```

Comment:
[question] Au fait... Je n'y ai pas fait attention avant mais dans "L'usage de cette route est **dépréciée**" ce n'est pas "déprécié" plutôt si le sujet c'est l'usage ?

Reply:
ah pas faux 😬 bon, tout est noté comme ça, je pense je vais laisser, c'est pas grave, ce sera plus facile de les retirer en faisant une recherche globale